phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widgets/lists/ListWidgetEntry.java

```diff
@@ -105,7 +117,58 @@ public ListWidgetEntry(E item, Optional<URI> iconUri, URI defaultIconUri, String
     }
 
     public static ListWidgetEntry<ContainerDTO> create(ContainerDTO container) {
-        return new ListWidgetEntry<>(container, Optional.empty(), StaticMiniature.CONTAINER_MINIATURE,
+        Optional<URI> shortcutMiniature = Optional.empty();
```

Comment:
You can remove this line now, so that you have 
```java
Optional<URI> shortcutMiniature = saveBufferedImage(segmentedMiniature, container.getName());
```
right?